nextflow.config

```
@@ -29,20 +25,19 @@ params {
     multiqc_methods_description = null
 
     // Boilerplate options
-    outdir                       = null
+    outdir                       = "null"
```

```suggestion
    outdir                       = null
```
This parameter is required, so it needs to be `null` by default